### DISCUSSION TITLE
User Agent can not  be modifed by setting.UserAgent

### DISCUSSION THREAD

**Discussion by jinshiyx1989** [cefsharp/CefSharp#4694] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

95.0.4638.69

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net 5

### Implementation

WinForms

### Reproduction Steps

// user the CefSharp.Winform.net core
CefSettings cefSettings = new CefSettings();
cefSettings.UserAgent = "Mozilla/5.0 (iPhone; CPU iPhone OS 10_0_1 like Max OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) Mobile/14A403 IHexin/10.20.10 (Royal Flush) ihexin/1234";
Cef.Initialize(cefSettings);

### Expected behavior

I expecte the  userAgent can been modified

### Actual behavior

I found that the UserAgent has not been truly modified. My CefSharp version is 95. The other two methods regarding the Agent in 'https://github.com/cefsharp/CefSharp/wiki/General-Usage#initialize-and-shutdown' are working fine

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
> Please only open an issue if you can reproduce the problem with version 120.1.80 or later.

As per the issue template. Please don't open issues for old **unsupported** versions. 

> Expected behavior
> I expecte the userAgent can been modified

The `UserAgent` can only be **set** once via `Cef.Initialize(settings)` and you can verify by loading `chrome://version` in ` ChromiumWebBrowser` instance.
